src/components/Button/index.examples.js

```diff
@@ -25,7 +25,7 @@ storiesOf('Button')
       <div className={btnDisp}>
         {standardButtonExamples.map((btn, i) => {
           return (
-            <Button type={btn.type} onClick={clickHandlerFixture}>
+            <Button type={btn.type} onClick={clickHandlerFixture} key={i}>
```

Comment:
Can't believe I missed `key`, I'm usually the one catching that for other people! :)